### PR TITLE
Feature/muwm 4324

### PR DESCRIPTION
**Pull request by fanglinfang**
Spec: https://docs.google.com/document/d/1Dg5nbbwmRrR3Nsy3FHMQf3OMTea-QVi93Rw54V3XgdM/edit#heading=h.rbg1dx6vb0pi

Test spec URL: https://docs.google.com/document/d/1MUY-Q-sH9Q14YE59Jgq8RbAvzUm7AuA1qhhi6cC4VZU/edit

Build on my-test.
To be release on 7/23 (Monday).

### CODE REVIEW

File: myuw/templates/handlebars/card/international_student.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "international_student_card_content" %}
+<div class="myuw-card">
+    <h3 class="myuw-card-title">
+        International Student
+    </h3>
+
+    <div class="myuw-card-section last">
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: Looks like this card is still empty. Will approve the PR when this is finished.

Comment (fanglinfang) [reply myuw/templates/handlebars/card/international_student.html]: just an empty template for now, not live. this card has been comment out on landing.js.

Comment (klpittman) [reply myuw/templates/handlebars/card/international_student.html]: ah ok, sounds good!

File: myuw/models/popular_link.py [hunk 2]
```diff
@@ -1,9 +1,24 @@
+import json
 from django.db import models
 
 
 class PopularLink(models.Model):
     affiliation = models.CharField(max_length=80, null=True)
     pce = models.NullBooleanField()
+    intl_stud = models.BooleanField(default=False)
```
Comment (fanglinfang) [inline myuw/models/popular_link.py]: Hi Stephen, 
Let me know if you prefer to have intl_stud  as one of the affiliation

File: myuw/static/js/landing.js [hunk 3]
```diff
@@ -72,6 +72,10 @@ var Landing = {
             TacomaApplicationCard
         ];
 
+        // if (window.user.intl_stud) {
```
Comment (devights) [inline myuw/static/js/landing.js]: Should this (and the mobile one below) be commented out?

Comment (fanglinfang) [reply myuw/static/js/landing.js]: No, that's for the next release.

**Pull request merged**